ParabolInc/action
# New auth branding

Fixes #1796 

(This currently includes commits from #1797, which will go away once that merges and this rebases on `master`)

## Of note

- I'm looking for feedback on not just how accurate these styles are, but how they're implemented. Component taxonomy, APIs, etc. are all in play.

## Testing

- Check out `new-auth-branding` locally
- Flip the `newSignIn` to `true`
- Test all the new auth UI
- Verify that it looks/feels/behaves correctly, according to the hi-fi wireframes
